recipes/javascripthon/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "javascripthon" %}
+{% set version = "0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 95fc362624513c5f0ec0ee0f355cf26807621da6fb94af94b0cf0ed313e4e957
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - pj = metapensiero.pj.__main__:main
+
+requirements:
+  host:
+    - python <3.8
```

Does this really not work with Python 3.8? I didn't see any issues upstream. I did notice that it needs to be at least 3.5:
```suggestion
    - python >=3.5
```
You should also edit `run` to match.

If you really need it limit to less than Python 3.8:
```suggestion
    - python >=3.5,<3.8
```

---

You're right, it _does_ seem to work with Py3.8, just did a quick test with some example code.